boards/st/steval_stwinbx1/steval_stwinbx1.dts

```diff
@@ -0,0 +1,330 @@
+/*
+ * Copyright (c) 2024 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+ #include <st/u5/stm32u585Xi.dtsi>
+ #include <st/u5/stm32u585aiixq-pinctrl.dtsi>
+ #include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "STMicroelectronics STEVAL-STWINBX1 Development kit";
+	compatible = "st,steval_stwinbx1";
+
+	chosen {
+		/*
+		 * By default, Zephyr console and shell are assigned to
+		 * USB CDC/ACM. To enable console and shell over UART,
+		 * uncomment the 2 following lines and set the correct
+		 * config in steval_stwinbx1_defconfig.
+		 *
+		 */
+		// zephyr,console = &usart2;
+		// zephyr,shell-uart = &usart2;
+		/*
+		 * To enable console and shell over UART,
+		 * comment the 2 following lines
+		 */
+		zephyr,console = &cdc_acm_uart0;
+		zephyr,shell-uart = &cdc_acm_uart0;
+
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		green_led_1: led_1 {
+			gpios = <&gpioh 12 GPIO_ACTIVE_HIGH>;
+			label = "LED_1";
+		};
+		orange_led_1: led_2 {
```

Comment:
could  `orange_led` be enough for labeling ?

Reply:
done